modules/Material/ApplicationWindow.qml

```diff
@@ -106,6 +106,11 @@ Controls.ApplicationWindow {
     }
 
     OverlayLayer {
```

Comment:
The tooltip layer should be above dialogs and dropdowns, so you can have tooltips in a dialog.